src/plugins/intel_npu/src/plugin/npuw/compiled_model.cpp

```diff
@@ -146,7 +147,7 @@ ov::npuw::CompiledModel::CompiledModel(const std::shared_ptr<ov::Model>& model,
         rewr.run_on_model(model);
     }
 
-    auto partitioning = getPartitioning(model, m_cfg);
+    auto partitioning = getPartitioning(model, m_cfg, m_weights_bank);
```

Comment:
hmm based on our latest decisions, partitioning shouldn't know about weight banks anymore?

Reply:
Correct, forgot to remove it

Reply:
Done